eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs

```diff
@@ -175,6 +175,12 @@ poolCertTransition ::
   PoolCert c ->
   TransitionRule (ledger era)
 poolCertTransition (PoolEnv slot pp) ps@PState {psStakePoolParams, psFutureStakePoolParams, psRetiring} = \case
+  -- FOR CONWAY
+  --
+  --   POOL-regpool : let open PParams pp ; open PoolParams poolParams in
+  --     c ∉ dom (pools ˢ)
+  --     ────────────────────────────────
+  --     pp ⊢ ⟦ pools , retiring ⟧ᵖ ⇀⦇ regpool c poolParams ,POOL⦈ ⟦ ❴ c , poolParams ❵ᵐ ∪ᵐˡ pools , retiring ⟧ᵖ
```

Comment:
I think the logic is actually the same in Conway, the spec just doesn't reflect every detail from the previous eras.

Reply:
Ok, a few things might actually change, but they're not in the spec yet

Reply:
I think it's better to just refer to the spec, since the comments can easily get outdated.